website/src/parsers/js/typescript.js

```diff
@@ -64,6 +65,14 @@ export default {
     }, compilerHost);
 
     const sourceFile = program.getSourceFile(filename);
+    
+    // workarounds issue described at https://github.com/Microsoft/TypeScript/issues/18062
+    for (const name of Object.keys(ts.SyntaxKind).filter(x => isNaN(parseInt(x)))) {
```

Comment:
This should happen in `loadParser` and not on each `parse` invocation.

Reply:
Fixed. Thanks!